README.md

```diff
@@ -38,15 +37,9 @@ To get a local copy up and running follow these simple example steps.
 
```

Comment:
Excellent work including a readme in your project. However, there are ways to make your readme look more professional. You can add the steps to Set up the repo and also I have noticed you have not added a title and introduction to the readme. Kindly fix it. So, anyone visiting your repo will not face problems understanding the Aim of the project.